src/Infrastructure/Elastic/ElasticClientBuilder.php

```diff
@@ -64,8 +65,14 @@ public static function fromConfig(Repository $config): ClientBuilder
             $builder->setSSLCert($path, $password);
         }
 
-        if($config->get('explorer.logging', false) && $config->has('explorer.logger')) {
-            $builder->setLogger($config->get('explorer.logger'));
+        if($config->get('explorer.logging', false) && $config->get('explorer.logger')) {
```

Comment:
Why the change in the second part of the if statement?

Reply:
You need both to be able to set the logger. If logger is not set you get a hard error because null is not an instance of logger